content/docs/1.6.4/deploy/important-notes/index.md

```diff
@@ -223,6 +224,16 @@ Longhorn v{{< current-version >}} does not allow the following:
 However, Longhorn still allows the listed actions when cloning or restoring volumes created with earlier Longhorn
 versions.
 
+### Backup Related Custom Resources Might Disappear
```

Comment:
After rethinking the title and description, not only the customer resources, we need to emphasize the **backup data in backup target** might be deleted. @mantissahz Can you create a PR for this? Thanks.